Home.md

```diff
@@ -78,7 +97,12 @@ db:schema:load` to load the database schema into each tenant.
 
 4.  Run `rake db:seed` on the development database,
 which creates a few special users, including the administrative user
-`admin@audience1st.com` with password `admin`.
+`admin@audience1st.com` with password `admin`. If you get this error, ignore it and proceed.  
```

Comment:
sorry this is so late.  why would this error occur?  seeding usually starts by dropping the whole db, so the admin user should not already exist....

Reply:
I'm not sure, this was behavior that the whole team observed. This occurs when setting up the app from a clean pull from github.